### ISSUE TITLE
Dictionary to store hurricane names

### ISSUE THREAD

**Issue by WxBDM** [WxBDM/PyNimbus#3]
See title - this would be used for `get_past_cyclone_info`. The primary purpose of this is to obtain information from past outlooks. Dictionary should be laid out as such:
```python
h_2019 = { "Andrea" : ["al01", "Atlantic"], ... , "Gil" : ["ep08", "East Pacific"], ... }
```
See the [NHC archive](https://www.nhc.noaa.gov/gis/archive_forecast.php?year=2019) for a full list. Previous years are needed as well.

**Comment by rishabmenon** [comment]
Hey! I would love to work on this! :)

**Comment by WxBDM** [comment]
Everything looks good per merge. One thing I didn't foresee is having 2 of the same name in the same year in both the Atlantic and EPAC basins. 2016 has this scenario, so we'll have to figure out a way around this. I'll leave this open if you want to work towards a solution to counteract this.

**Comment by rishabmenon** [comment]
Yes, I did notice it. This discrepancy occurs for another year as well! I'll be happy to work towards a solution around this.
